amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2092,6 +2097,417 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  ## Hold containers
+  '/hold-containers/{holdContainerId}/holds':
+    get:
+      tags:
+        - hold-containers
+      summary: Get holds
+      description: |
+        Returns a list of holds.
+
+        Minimal information for each child is returned by default.
+
+        You can use the **include** parameter (include=allowableOperations) to return additional information.
+      operationId: getHolds
+      parameters:
+        - $ref: '#/parameters/holdContainerIdParam'
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+        - $ref: '#/parameters/holdContainerEntryIncludeParam'
+        - $ref: '#/parameters/holdContainerIncludeSourceParam'
+        - $ref: '#/parameters/fieldsParam'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/TransferContainerAssociationPaging'
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to read **holdContainerId**
+        '404':
+          description: "**holdContainerId** does not exist"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    post:
+      tags:
+        - hold-containers
+      summary: Create holds
+      description: |
+        Creates a hold as a child of **holdContainerId**.
+
+        You must specify at least a **name** and the **"rma:holdReason"** property.
+
+        You can create a hold like this:
+        ```JSON
+        {
+          "name":"My Hold"
+          "properties":
+          {
+            "rma:holdReason":"My reason"
+          }
+        }
+        ```
+        You can set properties when creating a hold:
+        ```JSON
+        {
+          "name":"My Hold"
+          "properties":
+          {
+            "rma:holdReason":"My reason"
+            "cm:description":"Example description"
```

Comment:
This shouldn't be so generic. I would expect to see:
```
{
  "name": "My hold",
  "reason": "My reason",
  "description": "My description"
}
```
I didn't check what properties are usually found on holds, but you get the idea.